ansible/container-infra.yml

```diff
@@ -10,7 +10,7 @@
     os_container_infra_auth: "{{ openstack_auth }}"
     os_container_infra_auth_type: "{{ openstack_auth_type }}"
     os_container_infra_cluster_name: "{{ cluster_name }}"
-    os_container_infra_state: "{{ cluster_state | default('query') }}"
+    os_container_infra_state: "{{ cluster_state | default('present') }}"
```

Comment:
As someone not familiar with this, it would be helpful to explain why this change has been made in the commit message.

Reply:
This allows the behaviour to match with that of cluster-infra.
Ensuring state is present is a more common use case than query.

Reply:
I have expanded the commit message with this info to that end.